kiosk/src/Models/BuiltSimJsInfo.ts

```diff
@@ -0,0 +1,10 @@
+export class BuiltSimJSInfo {
+    constructor(
+        public js: string,
+        public targetVersion?: string,
+        public funArgs?: string[],
+        public parts?: string[],
+        public usedBuiltinParts?: string[]
+    ) {
+    }
+}
```

Comment:
```suggestion
export interface BuiltSimJSInfo {
    js: string;
    targetVersion?: string;
    funArgs?: string[];
    parts?: string[];
    usedBuiltinParts?: string[];
}
```

up to you / I don't have a strong preference, but interface does slightly more accurately represent it (only benefit would be that it would make sure nobody tries to add a method here and call it on something in `builtGamesCache`, but I'd describe that as 'very unlikely' so if you prefer it this way that's fine :))

Reply:
Actually alos happened to notice funArgs should be 
```
fnArgs?: { [a: string]: String[] }
```
re: https://github.com/microsoft/pxt/blob/master/localtypings/pxtarget.d.ts#L1019 but also doesn't matter much as we never actually read it here / just for context.

Reply:
Yeah, I was thinking about using an interface, too. I was just thinking about consistency with the other self-defined datatypes that we use in Kiosk. I might just change all of them to interfaces because I agree, class definitions don't make sense. Is that something I should include in this PR?

Reply:
Up to you, totally fine to split it into another cleanup one afterwards.

Same with second point, either way works for me so whatever is easiest for you~

Reply:
Okay, I'll just make a small PR for those changes. Thanks Joey!